### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by khoroshevskyi**
### Changes:
- Added user delete method
- Added history tables and methods while updating project

### TODO:

 - [x] Add changelog
 - [x] Update version

### CODE REVIEW

File: pepdbagent/modules/sample.py [hunk 1]
```diff
@@ -314,7 +313,7 @@ def delete(
                 parent_mapping = sample_mapping.parent_mapping
                 child_mapping = sample_mapping.child_mapping
                 session.delete(sample_mapping)
-                if parent_mapping:
+                if child_mapping:
```
Comment (nleroy917) [inline pepdbagent/modules/sample.py:316]: Was this a bug?

File: pepdbagent/modules/project.py [hunk 2]
```diff
@@ -591,6 +652,7 @@ def _update_samples(
         project_id: int,
         samples_list: List[Dict[str, str]],
         sample_name_key: str = "sample_name",
+        history_sa_model: Union[HistoryProjects, None] = None,
```
Comment (nleroy917) [inline pepdbagent/modules/project.py:655]: What is `sa_model` ? Should `sa` be something more descriptive?

**Review by nleroy917 — DISMISSED** [review]
👍🏼

File: pepdbagent/modules/sample.py (hunk 1 above)
Comment (khoroshevskyi) [reply pepdbagent/modules/sample.py:316]: yeah ...

**Pull request merged**